docs/gemstones/RL9_network_manager.md

```diff
@@ -128,7 +128,7 @@ You can also modify one or more properties of the connection directly from the c
 Shell > nmcli connection modify CONNECTION_NAME autoconnect yes ipv6.method dhcp
 ```
 
-!!! info
+!!! info "Info"
 
```

Comment:
Again, both of these admonitions should be left as-is. There is no need to make them "custom" admonitions. The capitalization will happen automatically when displayed. Again, [see here](https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/)